### ISSUE TITLE
SoftEther VPN

### ISSUE THREAD

**Issue by cpu** [jlund/streisand-discussions#19]
Suggested by @dimzon in https://github.com/jlund/streisand/issues/140

> https://www.softether.org/
> It's open source multi-protocol solution, claims is faster than native OpenVPN 
> ![Chart1](https://www.softether.org/@api/deki/files/4/=1.2.jpg?size=webview)
>
> ![Speed comparision](https://www.softether.org/@api/deki/files/12/=1.3.jpg)
> Also implements it's own protocol over HTTPS
>
> ![Feature comparision](https://www.softether.org/@api/deki/files/680/=comparison3.png)
>
> It will be great to have streisand-like project build on SoftEther VPN

**Comment by acwohlfarth** [comment]
Any update in this regard? Seems like an interesting open source addition.

**Comment by cpu** [comment]
@acwohlfarth No updates. Any updates should be reflected here so you're watching the right place :-)

@acwohlfarth Is there something in particular about SoftEther that makes you interested that the existing Streisand services don't offer?

**Comment by acwohlfarth** [comment]
For now the possibility to get around the Great Firewall of China appears interesting, since it is getting more and more sophisticated.

**Comment by cpu** [comment]
@acwohlfarth Do you find that Shadowsocks and the other options already packaged in Streisand are being blocked by the GFW? I have heard from users in China that find the existing options are working today.

The GFW is a moving target - we could spend time integrating SoftEther just to find the GFW blocks it a week or two after :-/

**Comment by mveplus** [comment]
There is one feature that is still not included by any other vpn's at present in Streisand, - MS-SSTP aka Microsoft native SSL support. Last time I've tried it showed mixed tunnel results in different regions in China, but it works great everywhere else. 
The other benefit from having SoftEther on-board is that we can minimize the services running - it can exchange OpenVPN and IPsec OpenSwan and L2TP running a single service that supports all these, which will reduce memory usage and can simplify general setup. 
SoftEther has built-in SoftEther client and admin for Windows and supports virtual bridges as well.

**Comment by sirkyuubi** [comment]
I have found that the service currently in Streisand are sufficient for getting passed the great firewall, but I also would like to see SoftEther as an option. As stated above it could consolidate a good portion of what Streisand offers into one package, reducing memory, and increase speed. Being in china I'm always very interested in anything that can increase stability, and lower latencey.

**Comment by Willy1216** [comment]
Can't use in China now.

**Comment by yrwyjz** [comment]
free vpn always is not stable in china, buy a [good one](https://git.io/JejnS)

**Comment by chipitsine** [comment]
hi, from the SoftEtherVPN project, I am glad to say that project is pretty alive.
me and @davidebeatrici can help managing process of integrating SE VPN to streisand

**Comment by trochdewei** [comment]
@chipitsine It would be nice to have easy way to setup SoftEther VPN Protocol on linux clients. Something like openvpn provides(systemd service files, scripts to handle dns). For now we need to manually adjust routing table
